ddtelemetry/src/metrics.rs

```diff
@@ -10,7 +10,7 @@ use std::{
 use ddcommon::tag::Tag;
 use serde::{Deserialize, Serialize};
 
-use crate::data;
+use crate::data::{self, metrics};
```

Comment:
Would have been nice to use either `data::metrics::...` or `metrics::...` and not mix both.